capstone/src/main/java/com/google/sps/servlets/scraperServlet.java

```diff
@@ -44,6 +47,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Elements restaurantNameElements = page.select(restaurantNameSelector);
 
       for (Element restaurantName : restaurantNameElements) {
+        String name = restaurantName.text(); 
+        details.request(detailedPlaces, name);
+
         restaurantNames.add(restaurantName.text());
```

Comment:
instead of passing back and forth the `detailedPlaces`, you could have `details.request()` return a `PlaceDetails` object and add it to `detailedPlaces`. This way could help minimize possible rep exposure through encapsulation :)

Reply:
Makes sense. Thanks for pointing that out